src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -202,6 +205,9 @@ CHIP_ERROR nl_Chip_Stack_Shutdown();
 const char * nl_Chip_Stack_ErrorToString(CHIP_ERROR err);
 const char * nl_Chip_Stack_StatusReportToString(uint32_t profileId, uint16_t statusCode);
 void nl_Chip_Stack_SetLogFunct(LogMessageFunct logFunct);
+
+CHIP_ERROR nl_Chip_GetDeviceByNodeId(chip::DeviceController::ChipDeviceController * devCtrl, chip::NodeId nodeId,
```

Comment:
Why do we have the `nl` prefix for all those methods ?

Reply:
Just to be consistent with other code, there will be another pr to change them to "chip_"